index.js

```diff
@@ -37,6 +39,17 @@ try {
 
     const vswhereResult = spawn(vswherePath, vswhereArgs, {encoding: 'utf8'})
     if (vswhereResult.error) throw vswhereResult.error
+
+    if (verbose) {
+      const args = [
+        '-nologo',
```

Comment:
The first three args are the same as in `vswhereArgs`; perhaps a separate prefix list of common options should be defined?